x-pack/qa/smoke-test-watcher-with-security/src/test/java/org/elasticsearch/smoketest/SmokeTestWatcherWithSecurityIT.java

```diff
@@ -286,16 +283,14 @@ public void testIndexActionInsufficientPrivileges() throws Exception {
         boolean conditionMet = objectPath.evaluate("hits.hits.0._source.result.condition.met");
         assertThat(conditionMet, is(true));
 
-        Response response = adminClient().performRequest("GET", "index_not_allowed_to_read/doc/my-id",
-                Collections.singletonMap("ignore", "404"));
+        Response response = adminClient().performRequest(new Request("HEAD", "/index_not_allowed_to_read/doc/my-id"));
         assertThat(response.getStatusLine().getStatusCode(), is(404));
     }
 
     private void indexWatch(String watchId, XContentBuilder builder) throws Exception {
-        StringEntity entity = new StringEntity(Strings.toString(builder), ContentType.APPLICATION_JSON);
-
-        Response response = client().performRequest("PUT", "_xpack/watcher/watch/" + watchId, Collections.emptyMap(), entity);
-        assertOK(response);
```

Comment:
did you remove this assertion intentionally?

Reply:
Yes. It just asserts that we return a 200 or 201. Since we through an exception on non-2xx replies anyway I figured it isn't useful.

Reply:
makes sense